pkg/process/checks/process.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/containers"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/gopsutil/cpu"
```

Comment:
Should this be removed too, or is it coming later?

Reply:
Yea this one is coupled a bit tightly in gopsutil, I need time to move it into procutil.